### PR TITLE
Add .pbs file, ran explore.py on Flux

### PR DESCRIPTION
**Pull request by at-tran**
Number of rows in the training set:     903653
Number of columns in the training set:  59
Number of rows in the test set:         804684
Number of columns in the test set:      57
The most visit times for a customer in train set is:  63315
95.00% of customers spend less than: $0.00
97.50% of customers spend less than: $0.00
99.00% of customers spend less than: $2718.00
99.90% of customers spend less than: $37980.83
99.99% of customers spend less than: $227456.03

The 6 most common sources of traffic are :
 google: 400788
 youtube.com: 212602
 (direct): 143028
 mall.googleplex.com: 66416
 Partners: 16411
 analytics.google.com: 16172

Channel Grouping Counts:
        Organic Search: 381561
        Social: 226117
        Direct: 143026
        Referral: 104838
        Paid Search: 25326
        Affiliates: 16403
        Display: 6262
        (Other): 120
Mean Total Revenue by Channel Grouping
        Display: $1251.00
        Referral: $621.37
        Direct: $304.03
        Paid Search: $171.99
        Organic Search: $85.54
        (Other): $8.32
        Affiliates: $3.64
        Social: $2.17

The transaction revenues by region are:
Maharashtra                    2.425614e+02
Metro Manila                   7.747378e+03
Dublin City                    8.188525e+03
North Holland                  1.242876e+04
Buenos Aires                   1.464111e+04
Istanbul                       1.617852e+04
Masovian Voivodeship           1.677920e+04
Lima Region                    2.118172e+04
Bangkok                        2.139318e+04
Bogota                         2.140061e+04
Delhi                          2.328344e+04
Gujarat                        2.524784e+04
Tel Aviv District              2.609964e+04
Karnataka                      3.491600e+04
County Dublin                  3.695068e+04
Attica                         3.818966e+04
Taipei City                    4.643462e+04
Bucharest                      5.308176e+04
Stockholm County               5.589286e+04
Prague                         7.415563e+04
Hamburg                        7.598566e+04
Tbilisi                        8.287805e+04
Kanagawa Prefecture            8.886154e+04
Vienna                         9.644695e+04
Ile-de-France                  1.103596e+05
England                        1.106228e+05
Zurich                         1.211298e+05
Indiana                        1.270909e+05
New South Wales                1.324615e+05
State of Sao Paulo             1.341299e+05
...                                     ...
Oregon                         8.943442e+05
Utah                           9.832258e+05
Tokyo                          1.049342e+06
not available in demo dataset  1.271398e+06
Minnesota                      1.373160e+06
Pennsylvania                   1.806549e+06
Nevada                         1.911752e+06
Missouri                       2.058062e+06
New Jersey                     2.086270e+06
Iowa                           2.183053e+06
North Carolina                 2.453425e+06
Ontario                        2.811494e+06
Texas                          3.275673e+06
Alberta                        3.455433e+06
California                     3.493169e+06
South Carolina                 3.756324e+06
Colorado                       3.987917e+06
Arizona                        4.105956e+06
District of Columbia           4.190588e+06
Nebraska                       4.513235e+06
Washington                     4.603500e+06
Georgia                        7.096050e+06
Virginia                       7.131867e+06
New York                       8.329000e+06
Massachusetts                  8.669727e+06
Illinois                       9.612773e+06
Michigan                       1.035626e+07
Tennessee                      1.135995e+07
Pichincha                      2.100941e+07
Zulia                          2.304556e+07

[79 rows x 1 columns]
85.96% of all visitors to the site visit exactly once.

### CODE REVIEW

File: googleanalytics.pbs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+####    or if you use software that is restricted to campus use.
+
+#PBS -A engin_flux
+#PBS -q flux
+
+#### #### ####  These are the least frequently changing options
+
+####  Your e-mail address and when you want e-mail
+
+#PBS -M uniqname@umich.edu
```
Comment (jonathancstroud) [inline googleanalytics.pbs]: Use `${USER}@umich.edu` if that works

Comment (TashaGospel) [reply googleanalytics.pbs]: I tried using `${USER}@umich.edu` but PBS seems to take that string literally, so it seems like we have to manually enter the email address
